### ISSUE TITLE
Log terraform version

### ISSUE THREAD

**Issue by NimJay** [GoogleCloudPlatform/cloud-foundation-toolkit#2321]
### Component

* The `cloud-foundation-toolkit` image used by Blueprint (GitHub repo) integration tests.

### TL;DR

* Cloud Build logs for Jump Start Solution (JSS) integration tests would ideally always include the version of terraform used by `gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS`.

### Detailed design

* Would it make sense to add a `terraform version` command here: https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/044c3ca90693449fd05d1e7e58229cbf263db864/infra/build/developer-tools/build/scripts/task_helper_functions.sh#L577

### Additional information

* For now, we could just run (inside our cloudbuild.yaml files) something like:
```
- id: log-terraform-version
  name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
  args: ['/bin/bash', '-c', 'terraform version']
```

**Comment by apeabody** [comment]
Logging the TF version would be useful. I would suggest we add to the `blueprint-test` `defaultInit` (or similar), which would have the advantage of not requiring an update to the repos/cloudbuild, and also benefit non-cloudbuild/cft cli use cases.

**Comment by apeabody** [comment]
@NimJay TF version logging will be included in https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/2334
